features/support/time_machine_helpers.rb

```diff
@@ -0,0 +1,41 @@
+# frozen_string_literal: true
+
+module TimeMachineHelpers
+  def time_machine(till)
+    travel_to(Time.zone.now)
```

Comment:
I'm wondering if this line be changed to `travel_back`, it would fix the Codeclimate warning, but not sure if it will change the behavior.

Reply:
It can't be replaced by `travel_back` because `travel_to` freezes the time and `travel_back` unfreezes the time, so different behavior. But I think it could be replaced by `freeze_time` which basically calls `travel_to(Time.now)`, let me check.